microbench/src/main/java/io/netty/microbench/channel/epoll/EpollSocketChannelBenchmark.java

```diff
@@ -51,7 +51,7 @@ public void setup() throws Exception {
             public void run() {
                 throw new AssertionError();
             }
-        }, 5, TimeUnit.MINUTES);
+        }, 30, TimeUnit.MINUTES);
```

Comment:
5 minutes was slightly too close to firing with two benchmarks.  Updated this to be a little farther out.